### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by jramorris**
expanded validation callback payloads

### CODE REVIEW

**Review by britainjs — COMMENTED** [review]
Looks good overall. Just a couple of comments.

File: source/includes/_reward_validation_callback.md [hunk 1]
```diff
@@ -51,10 +52,24 @@ When the Reward Validation Callback receives a status code that is not a 200 or
...
+};
 ```
 
+| Property   | Type          | Description                                                                                                                      |
+| :--------- | :------------ | :------------------------------------------------------------------------------------------------------------------------------- |
+| eventType     | string        | The event type that triggered the conversion.                                                                                 |
+| recipientType | string        | The role of the user receiving the reward, "advocate" or "friend".                                                            |
+| event         | object        | Details about the event that triggered the reward. Described in detail below.                                                 |
+| campaignId    | string        | The id of the campaign associated with the reward.                                                                            |
+| advocate      | object        | Details about the Advocate who earned the reward. Described in detail below.                                                  |
```
[the remark this answers is not in the document]
Comment (britainjs) [reply source/includes/_reward_validation_callback.md:70]: This didn't target right, but I was specifically referring to line 70.

File: source/includes/_reward_validation_callback.md [hunk 2]
```diff
@@ -103,6 +115,98 @@ Product details will be available in the `purchase.products` property of the req
...
+
+### event: custom
+
+Custom event callbacks include all information sent in the original event payload.
+
+| Property    | Type   | Description                                             |
+| :---------- | :----- | :------------------------------------------------------ |
+| createdOn      | ISO timestamp | The date and time the event was created.      |
+| myProperty     | my prop type  | My prop's definition.                         |
+| etc.           |               |                                               |
```
[the remark this answers is not in the document]
Comment (jramorris) [reply source/includes/_reward_validation_callback.md:157]: I think that's much better.  I can post in eng as well

File: source/includes/_reward_validation_callback.md [hunk 3]
```diff
@@ -103,6 +115,98 @@ Product details will be available in the `purchase.products` property of the req
...
+
+### Merchants prior to September 2021
```
[the remark this answers is not in the document]
Comment (jramorris) [reply source/includes/_reward_validation_callback.md:159]: I agree.  I assumed that was more of a placeholder until I could get another opinion on it

File: source/includes/_reward_validation_callback.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jramorris) [reply source/includes/_reward_validation_callback.md:70]: good call out

**Review by britainjs — APPROVED** [review]
LGTM

**Pull request merged**